Updates README for new release (#127)
RockefellerArchiveCenter/fluxx_exporter: README.md

# Fluxx Exporter Documentation

## Table of Contents

- [Overview](#overview)
- [Installation](#installation)
- [Configuration](#configuration)
  - [Configure a Fluxx Instance](#configure-a-fluxx-instance)
  - [Configure an Amazon S3 Bucket (Optional)](#configure-an-amazon-s3-bucket-optional)
  - [Configure Fluxx Tables and Fields](#configure-fluxx-tables-and-fields)
- [Export Grant Records](#export-grant-records)
  - [Create Export Jobs](#create-export-jobs)
  - [Add Filters](#add-filters)
- [Logging](#logging)
- [User Management](#user-management)
- [Contributing](#contributing)
- [License](#license)
- [Funding](#funding)

## Overview
The Fluxx Exporter is an open-source tool that integrates with the grants management system [Fluxx](https://www.fluxx.io/) to automate exports of select elements of grant records. The tool is built with Python and Django, and uses the Fluxx API to export data from a configured Fluxx instance. It can export both structured data from Fluxx database fields and files that are attached to grant records.

> Foundations use grants management systems (GMS) such as Fluxx to manage their grant making from the time a grantee begins an application, through the award process, and on to the grantees’ final reporting on activities. For most foundations, the GMS is the permanent system of record for all grant-related records. This tool allows grants administrators, information managers, and archivists to select and export grant information for internal or external uses including for long-term preservation, researcher access, and organizational learning and evaluation.

Reviewer: ```suggestion
Foundations use grants management systems (GMS) such as Fluxx to manage their grant making from the time a grantee begins an application, through the award process, and on to the grantee's final reporting on activities. For most foundations, the GMS is the permanent system of record for all grant-related records. This tool allows grants administrators, information managers, and archivists to select and export grant information for internal or external uses including for long-term preservation, researcher access, and organizational learning and evaluation.
```

2. Enter a descriptive name, s3 bucket name, and AWS credentials

You can configure additional S3 instances as needed.

### Configure Fluxx Tables and Fields

Fluxx tables and fields are the backend database information that power Fluxx cards on the frontend. Fluxx Exporter uses these tables and their associated fields to allow you to customize what information to export. 

#### Step 1: Download Fluxx Table API Documentation Pages

**Fluxx administrator access is required.**

1. Go to `{fluxx-instance-base-url}/api/rest/v2/doc` to access the API documentation for your Fluxx instance.
2. From the list of API documentation links, click "GrantRequest".
3. Save this grant request table documentation webpage as an HTML file on your computer.
4. Repeat this process for all Fluxx tables that contain data you would like to export, saving each HTML documentation file for use by Fluxx Exporter.

Note: You can explore the existing tables and their associated fields in `{fluxx-instance-base-url}/api/rest/v2/doc` to determine what data and fields you want to be able to export from Fluxx.

#### Step 2: Upload Fluxx Table API Documentation Pages

> Fluxx Exporter requires information from your backend table API documentation to complete exports. Follow these steps to import the HTML files you just downloaded from Fluxx.

Reviewer: ```suggestion
Fluxx Exporter requires information from your API documentation to configure tables and fields for export. Follow these steps to import the HTML files you just downloaded from Fluxx.
```
